stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Message.kt

```diff
@@ -90,7 +90,10 @@ public data class Message(
     @InternalStreamChatApi
     var channelInfo: ChannelInfo? = null,
 
-    @SerializedName("reply_to_message")
-    var replyTo: Message? = null
+    @SerializedName("quoted_message")
+    var replyTo: Message? = null,
```

Comment:
I thought about it, but still not sure because in UI we call this as the "reply" option. Seems "quoted" the only name for BE, we can keep this naming only in serialization part